media-libs/partio/partio-1.10.1.ebuild

```diff
@@ -0,0 +1,49 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{6,7,8,9} )
```

Comment:
Just checking: the tests (if they exist) passed on all these, but if they do not exist, at least the program seemed to work (ideally with something using said bindings? or whatever)

Reply:
I checked that I could compile each version, and created test.bgeo using makeline (with one of the versions), and opened it using partview with each version. I can see a collection of points in a straight line. Some of the tests like makecircle complain when they use default values, but appear to give valid results when checked using partview.

Tests like testmerge and testcluster only pass if the source is still unpacked as they need src/data/*.bgeo to exist and need portage permissions. I think the tests should eventually be packaged to run only with FEATURES="test" and not be installed on the user's system.